e2e/specs/st_reuse_label.spec.ts

```diff
@@ -23,6 +23,12 @@ describe("reuse widget label", () => {
   });
 
   it("reuses a widget label for different widget types", () => {
+    cy.get('.stSlider [role="slider"]').should("not.exist");
```

Comment:
Can I take a look at this test fix before you merge? Not sure if this is correct

Reply:
Sure, if you take a look the first time the test does not have the slider, when you click on the second pass the slider is being shown